### ISSUE TITLE
Start year not getting updated

### ISSUE THREAD

**Issue by dipinknair** [ansys/pre-commit-hooks#125]
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

Start year not getting updated once you modify to year less  than the current and change it back to original year.

### 📝 Steps to reproduce

1. update to ``--start_year=2023 ``
2. run pre-commit and check files are updated
3. change to  ``--start_year=2022`` 
 4. run pre-commit and check files are updated
 5. change back to ``--start_year=2023 ``
 6. Files are still showing 2022

### 💻 Which operating system are you using?

Windows

### 📀 Which ANSYS version are you using?

2024

### 🐍 Which Python version are you using?

3.11

### 📦 Installed packages

```shell
ansys-tools-path==0.3.2
build==1.0.3
cfgv==3.4.0
click==8.1.7
colorama==0.4.6
distlib==0.3.7
filelock==3.12.4
identify==2.5.30
importlib-metadata==6.8.0
iniconfig==2.0.0
nodeenv==1.8.0
packaging==23.2
platformdirs==3.11.0
pluggy==1.3.0
pre-commit==3.5.0
pyproject_hooks==1.0.0
pytest==7.4.3
PyYAML==6.0.1
virtualenv==20.24.6
zipp==3.17.0
```

### Note

Before contributing and starting any work, please make sure you agree with the [CLA](https://developer.ansys.com/form/cla-acceptance) and sign it in your PR.

**Comment by klmcadams** [comment]
This is because reuse takes the min of the years list. For example, if your file headers have "2022 - 2024" and you switch the start year to 2023, then the year list would be this: [2022, 2024, 2023, 2024], so the min would still be 2022

https://github.com/fsfe/reuse-tool/blob/1e785a4e0c3c852537072c56f16903b53dbd85f5/src/reuse/_util.py#L388

**Comment by SMoraisAnsys** [comment]
@klmcadams Could this be associated to hacktoberfest ?

**Comment by RobPasMue** [comment]
> @klmcadams Could this be associated to hacktoberfest ?

Nice idea! :)

**Comment by klmcadams** [comment]
Closed by #236
